modules/homa-lite/src/lib.rs

```diff
@@ -138,11 +157,17 @@ pub mod module {
 	#[pallet::getter(fn current_batch)]
 	pub type CurrentBatch<T: Config> = StorageValue<_, EraIndex, ValueQuery>;
 
-	/// The account in which the staking currency goes into to be transferred to the Relay chain.
-	/// RelayChainStashAccount: value: stash_account: AccountId
+	/// The cap on the total amount of staking currency allowed to mint Liquid currency.
+	/// StakingCurrencyMintCap: value: mint_cap: Balance
+	#[pallet::storage]
+	#[pallet::getter(fn staking_currency_mint_cap)]
+	pub type StakingCurrencyMintCap<T: Config> = StorageValue<_, Balance, OptionQuery>;
```

Comment:
`ValueQuery` should be fine. The default value `zero` is reasonable in this context, users cannot mint if cap not set.